text/text.md

```diff
@@ -1464,48 +1464,48 @@ than their will to take proper care of the circus property.
 Clown act
 -------------
 
-A few hundred yards down the path  the clown car sped past us,
-apparently also in pursuit of the attackers in their
+A few hundred yards down the path, the clown car sped past us,
+apparently also in pursuit of the attackers on their
 horses. Wigs and big shoes were coming out of its windows and they were honking
 and laughing and shooting all at the same time. It was not going to take long
 for them to reach our attackers and in due time we could see light 
 arising from somewhere in the path, a few hundred yards ahead of us. 
 
-I would have stopped right there but didn't, afraid of being uncovered
+I would have stopped right there, but didn't, afraid of being uncovered
 by the other circus hands. Well, you know, I mean uncovered as not
 being part of the circus brotherhood. Nobody could have a hint of what
-I intended to do, I hadn't tell no one. 
+I intended to do, I hadn't told no one. 
 
 Not that I was under a cover, mind you. I just didn't want them to
-know that my purpose in life was not to live in and by the circus,
-just to get something out of it.
+know that my purpose in life was not to live in and around the circus,
+instead it was just to get something from it.
 
 But I should have left everything right there and then, because I didn't like
-what I saw down the road. It was not nice, you know. Not that they
+what I saw down the road. It wasn't pretty, you know. Not that they
 didn't deserve it, but one thing is thinking that somebody deserves
-what comes his way and another wanting him to be done in front of
+what comes his way and another is wanting him to be done in front of
 you. 
```

Comment:
There was probably something wrong with the sentence, but adding "is"... I don't like it.